### PR TITLE
get started guide

### PR DESCRIPTION
**Pull request by xmonader**
### Related issues
- #180

### CODE REVIEW

File: docs/Get_Started_Guide.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
+# How to get started
+
+## To Run The Project follow these steps :
+
+- clone the Repository : `git clone https://github.com/threefoldtech/tfgrid_dashboard.git`
+- navigate to the Repository : `cd tfgrid_dashboard`
+- make your on branch : `git checkout -b development_<FEATURE_NAME>`
+- install the dependencies : `yarn install`
+- run the project : `yarn run`
```
Comment (rawdaGastan) [inline docs/Get_Started_Guide.md]: can you change `yarn run` to `yarn serve`

File: docs/Get_Started_Guide.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+- open your browser and navigate to http://localhost:3000/
```
Comment (rawdaGastan) [inline docs/Get_Started_Guide.md]: the port is 8080 ?

File: docs/Get_Started_Guide.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+- the router is the main component of the project it is responsible for routing the user to the right component
+- the router can have nested routes to navigate to different components
+
+##### How to use the router?
+
+- simply import the component that you want to route to
+- then append it to the routes array in this form :
+  ```
+  {
+    component: <Component.>,
```
Comment (rawdaGastan) [inline docs/Get_Started_Guide.md]: remove extra . in <Component.> :D

File: docs/Get_Started_Guide.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+    path: "/<path>",
+    children: <Nested-Router>,
+  }
+  ```
+- If you want to create new Category in the sidebar then you need to create a new component and add it to the router
+
+#### If you want to create new Category in the sidebar
+
+- you create a new folder with the name of the category and inside it components,router (these will be the nested routes),Assets,lib (for manipulative functions) folder
+- example :
```
Comment (rawdaGastan) [inline docs/Get_Started_Guide.md]: you can mention that it's explorer example

**Review by rawdaGastan — CHANGES_REQUESTED** [review]

**Review by MohamedElmdary — APPROVED** [review]

**Review by rawdaGastan — APPROVED** [review]

**Pull request merged**